Data/Sys/GameSettings/GK2.ini

```diff
@@ -2,3 +2,11 @@
 
 [Core]
 MMU = True
+
+[ActionReplay]
+$16:9 Widescreen [Vague Rant]
+0446A740 3FEDCD7C
+044795A4 3FEDEC81
```

Comment:
Are you sure this code works on all versions of the game? If not, it should be in a six-character INI file.

Reply:
Discussed in Discord, we believe that this should work in all versions.